swan-lake/development-tutorials/deployment-guide/containerized-deployment.md

```diff
@@ -514,7 +517,291 @@ Follow the steps below to execute the Kubernetes service.
 >**Note:** You can visit the <a href="https://github.com/ballerina-platform/ballerina-spec/blob/master/c2c/code-to-cloud-spec.md" target="_blank">Code to Cloud specification</a> for detailed information about all the supported features. As mentioned in the beginning, Code to Cloud is a tool created to make the development process easier. It does not cover the operational properties in Kubernetes. If you want to have these additional features in your deployment, or you need to change the generated Kubernetes artifacts, you can use <a href="https://kustomize.io" target="_blank">Kustomize</a> for modifying the generated YAML.
 
 
-## Kubernetes deployment with Kustomize
+## Openshift deployment
+
+### Create the Ballerina package
+Below sample describes a Ballerina application that reads a greeting string from a config map and greets the user upon HTTP request. By following the steps below, you can make a Kubernetes deployment that has container resource limits, horizontal pod autoscaling, config maps and liveness, readiness probes with the help of Code to Cloud.
+
+1. Execute the `bal new hello_openshift` command to create a new package named `hello_openshift` and go inside that directory.
+
+2. Replace the content of the `main.bal` file with the content below.
+
+   ***main.bal***
+
+    ```ballerina
+    import ballerina/http;
+
+    listener http:Listener helloEP = new (9090);
+
+    configurable string greeting = ?;
+
+    service /helloWorld on helloEP {
+        resource function get sayHello() returns string|error? {
+            return greeting + ", Openshift!";
+        }
+    }
+    ```
+2. Create `probes.bal` with the following contents.
+
+   ***probes.bal***
+
+    ```ballerina
+    import ballerina/http;
+
+    listener http:Listener probeEP = new (9091);
+    
+    service /probes on probeEP {
+        resource function get healthz() returns boolean {
+            return true;
+        }
+        resource function get readyz() returns boolean {
+            return true;
+        }
+    }
+    ```
+3. Create the `Config.toml` file in the package directory and paste the following content into it.
+
+   ***Config.toml***
+    ```toml
+    greeting = "Hello"
+    ```
+
+4. Add `cloud = "openshift"` under the `[build-options]` table into the `Ballerina.toml` file in the package.
+
+   ***Ballerina.toml***
+
+    ```toml
+    [build-options]
+    cloud = "openshift"
+    ```
+
+5. Create a file named `Cloud.toml` in the package directory and add the content below. You can use some properties from the docker sample here since `cloud="k8s"` option builds both Docker image and Kubernetes. You can see a brief description of the properties in the comments.
```

Comment:
```suggestion
5. Create a file named `Cloud.toml` in the package directory and add the content below. You can use some properties from the Docker sample here since the `cloud="k8s"` option builds both Docker image and Kubernetes. You can see a brief description of the properties in the comments.
```